certified-connectors/ActiveCampaign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,132 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Enable integrations with your ActiveCampaign account.",
+    "version": "1.0.0",
+    "title": "ActiveCampaign",
+    "x-ms-api-annotation": {
+      "status": "Preview"
+    },
+    "contact": {
+      "name": "Microsoft",
+      "email": "ci-3rdpartycontact@microsoft.com"
```

Comment:
@mmchome Please add a "url" property in the contact section. this should be your support contact page.
Thank you.

Reply:
done